### PR TITLE
Added info about the values of the HTTP_PROXY and HTTPS_PROXY var

### PR DESCRIPTION
**Pull request by dcaro**
Added some additional information pulled from .NET httpClient configuration: https://docs.microsoft.com/en-us/dotnet/api/system.net.http.httpclient.defaultproxy?view=net-5.0

### CODE REVIEW

File: docs-conceptual/azps-6.3.0/az-powershell-proxy.md [hunk 1]
```diff
@@ -28,3 +28,7 @@ The environment variables used are:
 
 On systems where environment variables are case-sensitive, the variable names may be all lowercase
 or all uppercase. The lowercase names are checked first.
+
+The proxy server may be a hostname or IP address, optionally followed by a colon and port number, or
+it may be an http URL, optionally including a username and password for proxy authentication. The
+URL must be start with http, not https, and cannot include any text after the hostname, IP, or port.
```
Comment (mikefrobbins) [inline docs-conceptual/azps-6.3.0/az-powershell-proxy.md:34]:
```suggestion
The proxy server may be a hostname, IP address, or HTTP URL, optionally followed by a colon and port number. Username and password are also optional for proxy authentication. The
URL must start with HTTP, not HTTPS, and cannot include any text after the hostname, IP address, or port number.
```

**Review by mikefrobbins — CHANGES_REQUESTED** [review]
See the suggested changes.

File: docs-conceptual/azps-6.3.0/az-powershell-proxy.md (hunk 1 above)
Comment (dcaro) [reply docs-conceptual/azps-6.3.0/az-powershell-proxy.md:34]: I do not know if the environment variable is case sensitive so I would prefer using `http` over HTTP.

**Comment by dcaro** [comment]
@mikefrobbins, thanks for the suggestion. I have update the text, however I prefer `http` over HTTP unless this conflict with docs guidance.

**Review by mikefrobbins — APPROVED** [review]
LGTM

**Pull request merged**